LondonBritishLibrary/orient/BLorient809.xml

```diff
@@ -104,13 +81,467 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            ፀዓቶሙ፡ ፪፻፡ ፲ወ፭፡ <sic>ዓመ፡</sic></q>
+                              </item>
+                              <item xml:id="a3">
+                                <locus target="#4v"></locus>
+                                <desc type="GuestText">The number of descendants
+                                  of Japhet, Shem and Ham.
+                                  <q xml:lang="gez">ውሉደ፡ ያፌት፡ ፲ወ፭ ነገድ፡ ወመክፈልቶሙ፡ ኮነ፡ ምድረ፡ ፀጋም፡ ወመንፈቀ፡
+                                  ምዕራብ፡ ወውሉደ፡ ሴምኒ፡ ፳ወ፭፡ ነገድ፡ ወመክፈልቶሙ፡ ኮነ፡ ምድረ፡ ጽባሕ፡ ወመንፈቀ፡ አዜብ፡ ወደቂቀ፡
+                                ካምኒ፡ ፴ወወ፪ ነገድ፡ ወመክፈልቶሙ፡ ኮነ፡ ምድረ፡ አዜብ፡ ወመንፈቀ፡ ምዕራብ፤ ወኮኑ፡ ኵሉ፡ ውሉደ፡ ኖኅ፡
+                              ፸ወ፪፡ ወእንበይነ፡ ዝንቱ፡ ተከፍሉ፡ ኀበ፡ ፸ወ፪፡ ልሳናት፡</q>
+                                </desc>
+                              </item>
+                              <item xml:id="a4">
+                                <locus target="#4v"></locus>
+                                <desc type="GuestText">Short glossary.</desc>
+                                <q xml:lang="gez"><seg part="I">በጻሕራት፡ በምጽዓድ፡ ብሂል፡ መቃጽው፡ ዘ፡ ምንቸቶች፡ በገዳመ፡
+                                አሐዱስ፡ ዘ፡ ስመ፡ ሀገር፡ ዘውእቱ፡ መጌጾ፡ አቀተውሙ፡ ዘ፡ አስገበራቸው፡ ውስተ፡ መቀረቡ፡
+                              ዘ፡ መቃረብያ፡ ተጠከዩ፡ ዘ፡ ጠፉ፡ ዐስበ፡ ቀፈት፡ ዘ፡ ዓሥራ፡ ሐዶን፡ ይሜኵዩ፡</seg></q>
+                              </item>
+                              <item xml:id="e3">
+                                <locus target="#2r #2v #4v #5r #5v"></locus>
+                                <desc>Short texts and scribbles, some of which are written
+                                upside down, partially erased.</desc>
+                                <q xml:lang="gez"></q>
+                              </item>
+                              <item xml:id="e4">
+                                <locus target="#2r"></locus>
+                                <desc type="StampExlibris">A red seal impression of the British Museum different from
+                                <ref target="#e2"></ref>.</desc>
+                              </item>
+                              <item xml:id="e5">
+                                <locus target="#4v"></locus>
+                                <desc type="StampExlibris">A red seal impression of the British Museum same as
+                                <ref target="#e2"></ref>.</desc>
+                              </item>
+                            </list>
+                          </additions>
+
+                      </physDesc>
+                    </msPart>
+                    <msPart xml:id="p3">
+                      <msIdentifier></msIdentifier>
+                      <msContents>
+                        <summary></summary>
+                        <msItem xml:id="p3_i1">
+                          <locus from="7ra" to="123v"></locus>
+                          <title type="incomplete" ref="LIT1011Abusak"></title>
+                          <textLang mainLang="gez"></textLang>
```

Comment:
maybe add otherLangs Amharic, if there are Amharic parts throughout? Or do you think they constitute a later addition?

Reply:
I think those Amharic commentaries are of a later origin, not attested in other mss as well